plugins/nirmata/nctl.go

```diff
@@ -0,0 +1,25 @@
+package nirmata
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func NirmataCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Nirmata CLI",
+		Runs:    []string{"nctl"},
+		DocsURL: sdk.URL("https://nirmata.com/docs/cli"), // TODO: Replace with actual URL
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
```

Comment:
Let's opt out of authentication for `nctl login` as well. 😄

Reply:
yess getit!!

Reply:
Thanks! I'm not familiar with the Nirmata CLI, are there any other commands which do not require authentication?

Reply:
I believe `version` does not require authentication, but could you confirm the rest, @itsCheithanya?